xml/cl.xml

```diff
@@ -6764,7 +6764,7 @@ server's OpenCL/api-docs repository.
                 <enum name="CL_QUEUE_COMPUTE_UNIT_LIMIT_ARM"/>
             </require>
         </extension>
-        <extension name="cl_ext_cxx_for_opencl" revision="0.0.0" supported="opencl">
+        <extension name="cl_ext_cxx_for_opencl" revision="1.0.0" depends="CL_VERSION_3_0,CL_VERSION_2_0+cl_khr_extended_versioning" supported="opencl">
```

Comment:
Can you please explain what it means, to have 2 different versions of core in the `depends` attribute?

Given coma, IG it means this extension can be supported if either:
- OpenCL is 2.0 and `cl_khr_extended_versioning` is supported.
- OpenCL is 3.0 is supported.

This is my best guess, but this is the first time coma was used in `depends` and `registry.rnc` explanation is still pretty surface-lvl:
```
# Boolean expression of core version and extension names using (),+ operators
Depends = attribute depends { text }
```
Before I thought the coma here is not an operator but a separator between `()` and `+` operators...

---

Also below:
```
        <extension name="cl_ext_float_atomics" revision="1.0.0" depends="CL_VERSION_2_0,CL_VERSION_3_0" supported="opencl">
```
Wouldn't `CL_VERSION_3_0` here just be redundant?